CMakeLists.txt

```diff
@@ -156,6 +157,11 @@ target_include_directories(xgboost
   INTERFACE
   $<INSTALL_INTERFACE:${CMAKE_INSTALL_PREFIX}/include>
   $<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}/include>)
+find_library(LIBUTIL NAMES util PATHS "/usr/lib64")
```

Comment:
Why do we need libutil? Is it necessary for Arrow? Can we make libutil optional?

Reply:
Removed libutil from CMakeLists.txt